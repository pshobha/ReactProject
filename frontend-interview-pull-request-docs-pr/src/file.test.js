import React from 'react';
import renderer from 'react-test-renderer';
import File from './file.component.js';

describe(`<File />`, () => {
	it(`renders a file with file name, date added, and username`, () => {
		const tree = renderer.create(<File
			indentationLevel={0}
			file={{
				id: 'file1',
				name: 'Genghis-Khan.png',
				dateAdded: 1489183017615,
				addedBy: {
					id: 'user1',
					name: 'Erich Ludendorff',
				}
			}}
			/>
		);
		expect(tree).toMatchSnapshot();
	});
});
