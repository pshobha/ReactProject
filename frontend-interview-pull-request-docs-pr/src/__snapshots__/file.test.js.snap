// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<File /> renders a file with file name, date added, and username 1`] = `
<div>
  <div
    className="row"
  >
    <div
      style={
        Object {
          "borderLeft": "0px solid deepskyblue",
        }
      }
    >
      Genghis-Khan.png
    </div>
    <div>
      3/10/2017
    </div>
    <div>
      Erich Ludendorff
    </div>
  </div>
</div>
`;
